Kernel/Devices/Storage/USB/BulkSCSIInterface.h

```diff
@@ -58,6 +58,75 @@ struct CommandStatusWrapper {
 };
 static_assert(AssertSize<CommandStatusWrapper, 13>());
 
+enum class SCSIDataDirection {
+    DataToDevice,
+    DataToHost,
```

Comment:
nit: SCSI calls those the initiator and target (https://en.wikipedia.org/wiki/SCSI_initiator_and_target).

Reply:
Changed it accordingly